src/messages/GE.json

```diff
@@ -473,5 +473,9 @@
   "This token is not a jwt token and cannot be decoded by jwt.ms": "This token is not a jwt token and cannot be decoded by jwt.ms",
   "The URL must contain graph.microsoft.com": "The URL must contain graph.microsoft.com",
   "Fetching code snippet failing": "Retry again",
-  "Fetching permissions failing": "Retry again"
+  "Fetching permissions failing": "Retry again",
+  "Reload consent-type": "Confirm that you have consented to Directory.Read.All or Directory.ReadWrite.All or DelegatedPermissionGrant.ReadWrite.All and that you have reauthenticated if you revoked this permission earlier then click Reload",
+  "An error occured during the revoking process": "An error occured during the revoking process",
+  "Retrying": "Retrying",
+  "You require the following permissions to read": "You require Directory.Read.All or Directory.ReadWrite.All or DelegatedPermissionGrant.ReadWrite.All to read permission grants for your tenant"
```

Comment:
Is this error only for the provided permissions?

Reply:
I don't understand the question, but this error is shown when a user attempts to load permission consent-types when they do not have the required permissions.

Reply:
The message seems to say
```
You require Directory.Read.All or Directory.ReadWrite.All or DelegatedPermissionGrant.ReadWrite.All to read permission grants for your tenant
```
there are 3 permissions there

Reply:
Yes. These are very specific to reading permission grants for the tenant.